src/Adapters/Storage/Postgres/ClientCreator.js

```diff
@@ -0,0 +1,31 @@
+const pgp = require('pg-promise')();
```

Comment:
This isn't the client. It's just pg-promise being initialized. A call to `pgp()` is what creates the client.